@Jasstkn , добрый день.
Я исправил имя Pod. Тем не менее получил ошибку
Вы могли бы подсказать, какой Pod не может запуститься?
```
==================================== ERRORS ====================================
__________________ ERROR at setup of test_resource_existence ___________________
additional-tests/test_110_frontend_pod.py:16: in frontend_pod
    p.wait_until_containers_start(timeout=60)
/tmp/pytest/lib/python3.6/site-packages/kubetest/objects/pod.py:289: in wait_until_containers_start
    utils.wait_for_condition(condition=wait_condition, timeout=timeout, interval=1)
/tmp/pytest/lib/python3.6/site-packages/kubetest/utils.py:114: in wait_for_condition
    .format(timeout, condition)
E   TimeoutError: timed out (60s) while waiting for condition <Condition (name: all pod containers started, met: False)>
____________________ ERROR at setup of test_resource_status ____________________
additional-tests/test_110_frontend_pod.py:16: in frontend_pod
    p.wait_until_containers_start(timeout=60)
/tmp/pytest/lib/python3.6/site-packages/kubetest/objects/pod.py:289: in wait_until_containers_start
    utils.wait_for_condition(condition=wait_condition, timeout=timeout, interval=1)
/tmp/pytest/lib/python3.6/site-packages/kubetest/utils.py:114: in wait_for_condition
    .format(timeout, condition)
E   TimeoutError: timed out (60s) while waiting for condition <Condition (name: all pod containers started, met: False)>
```

--- a/kubernetes-intro/frontend-pod-healthy.yaml
+++ b/kubernetes-intro/frontend-pod-healthy.yaml
@@ -6,7 +6,7 @@ metadata:
   name: frontend
 spec:
   containers:
-  - image: "dmizverev/otus-frontend:1.0.0"
+  - image: "dmizverev/otus-frontend:1.1.0"
     name: frontend
     env:
       - name: PORT

